src/emulator/storage/rules/manager.ts

```diff
@@ -1,52 +1,72 @@
 import * as chokidar from "chokidar";
-import * as fs from "fs";
 import { EmulatorLogger } from "../../emulatorLogger";
 import { Emulators } from "../../types";
-import { FirebaseError } from "../../../error";
 import { SourceFile } from "./types";
 import { StorageRulesIssues, StorageRulesRuntime, StorageRulesetInstance } from "./runtime";
+import { RulesConfig } from "..";
 
 /**
- * Loads and maintains a {@link StorageRulesetInstance} for a given source file. Listens for
- * changes to the file and updates the ruleset accordingly.
+ * Keeps track of the rules source file and maintains a generated ruleset for one or more storage
+ * resources.
+ * */
+export interface StorageRulesManager {
+  /** Sets source file for each resource using the rules previously passed in the constructor. */
+  start: () => Promise<StorageRulesIssues>;
```

Comment:
Consider using named interface functions here:
```ts
export interface StorageRulesManager {
  start(): Promise<StorageRulesIssue>;
  getRuleset(resource: string): StorageRulesetInstance | undefined;
  // etc
}
```

While not strictly required by the style guide AFAICT, in my opinion it's a lot more readable

Reply:
Done.